### PR TITLE
[1.13] Revert containers/storage to older version

### PR DESCRIPTION
**Pull request by mrunalp**
We saw some locking issues in the latest update which need more time to be sorted out.

### CODE REVIEW

**Comment by ashcrow** [comment]
Travis failure:
```
/home/travis/.gimme/versions/go1.11.8.linux.amd64/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-124318063/000008.o: unrecognized relocation (0x2a) in section `.text'
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
```

**Comment by mrunalp** [comment]
@ashcrow Yes, we suspect a travis toolchain issue since they have updated to 1.11.8.

**Comment by mrunalp** [comment]
Added a commit to switch travis to 1.10.x since one of the 1.10 jobs passed.

**Comment by nalind** [comment]
LGTM

File: vendor.conf [hunk 1]
```diff
@@ -19,7 +19,7 @@ github.com/sirupsen/logrus v1.0.0
 github.com/containers/image 93bced01015eb94bec4821df1876314be8197680
 github.com/docker/docker-credential-helpers d68f9aeca33f5fd3f08eeae5e9d175edf4e731d1
 github.com/ostreedev/ostree-go master
-github.com/containers/storage v1.11
+github.com/containers/storage 06b6c2e4cf254f5922a79da058c94ac2a65bb92f
```
Comment (vrothberg) [inline vendor.conf]: Why not moving to the lastest storage v1.12.x? That's already in master and seems to work.

**Pull request merged**